src/lib/phdi-building-blocks/phdi_building_blocks/standardize.py

```diff
@@ -1,93 +1,99 @@
-                        "valueBoolean": raw_family != std_family,
-                    }
-                )
-                name["family"] = std_family
-
-            if "given" in name:
-                std_givens = [standardize(g) for g in name["given"]]
-                raw_givens = [g for g in name["given"]]
-                any_diffs = any(
-                    [raw_givens[i] != std_givens[i] for i in range(len(raw_givens))]
-                )
-                patient["extension"].append(
-                    {
-                        "url": "http://usds.gov/fhir/phdi/StructureDefinition/given-name-was-standardized",  # noqa
-                        "valueBoolean": any_diffs,
-                    }
-                )
-                name["given"] = std_givens
+        # Given names are stored in a list, as there could be multiple,
+        # process them all and take the overall diff for metrics
+        if "given" in name:
+            transformed_names = [
+                standardize_text(g, **kwargs) for g in name.get("given", [])
+            ]
+            name["given"] = transformed_names
 
-    return bundle
+    return None
 
 
 def country_extractor(
-    resource: dict, code_type: Literal["alpha_2", "alpha_3", "numeric"] = "alpha_2"
+    patient: dict, code_type: Literal["alpha_2", "alpha_3", "numeric"] = "alpha_2"
 ) -> List[str]:
     """
-    Given a FHIR resource yield a generator containing all of the counries found in the
-    resource in a standard form sepcified by code_type.
-
-    We currently only handle extracting countries from patient resources, however as
-    need arises is the future funcionality can be expanded.
+    Given a patient, build a list containing all of the counries found in the
+    patient's addresses in a standard form sepcified by code_type.
 
-    :param dict resource: A FHIR resource respresented as a dictionary.
-    :param str code_type: A string equal to 'alpha_2', 'alpha_3', or 'numeric' to
-    specify which type of standard country identifier to generate.
-    :retrun list countries: A list containing country codes as specified by code_type
-    for each country found in resource.
+    :param patient: A patient from a FHIR resource
+    :param code_type: A string equal to 'alpha_2', 'alpha_3', or 'numeric' to
+        specify which type of standard country identifier to generate
     """
     countries = []
-    resource_type = resource.get("resource").get("resourceType")
-    if resource_type == "Patient":
-        for address in resource.get("resource").get("address"):
-            country = address.get("country")
-            countries.append(standardize_country(country, code_type))
+    for address in patient.get("address"):
+        country = address.get("country")
+        countries.append(standardize_country(country, code_type))
     return countries
 
 
-def standardize_phone(raw: str, countries: List = [None, "US"]) -> str:
```

Comment:
I disagree with the choice to rename this function. as it does more than just standardize a phone number's country. The call on line 131 puts the entire phone number in the E164 international standard. In fact the country standardization is partly optional if the user use the default value of `countries`

Reply:
After looking at the code again, I think I'm going to mark this one as resolved. There doesn't seem to be a whole lot of semantic difference between the two function names, and while it's true that `standardize_phone` is perhaps more descriptive of the resulting international standard result, the actual usage difference there is pretty negligible given that standardization into the international format will likely come about at least in part due to a country code.